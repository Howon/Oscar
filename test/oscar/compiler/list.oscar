def main() => unit = {
  Println(list<int>[]);
  Println(list<int>[1, 2, 3, 4, 5]);
  Println(list<string>["hey", "hi", "hello"]);
}
