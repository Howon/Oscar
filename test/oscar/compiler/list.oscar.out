[]
[1,2,3,4,5]
["hey", "hi", "hello"]
